backend/server.js

```diff
@@ -0,0 +1,62 @@
+const cors = require('cors');
+const morgan = require('morgan');
+const express = require('express');
+const SequelizeStoreInit = require('connect-session-sequelize');
+const session = require('express-session');
+const Sequelize = require('sequelize');
+const app = express()
+const PORT = 3000
+
+app.use(cors({
+  origin: 'http://localhost:5173',
+  credentials: true
+}));
+app.use(express.json())
+app.use(morgan())
+
+const newSequelize = new Sequelize('event_finder', 'postgres', '1025323', {
+    host: 'localhost',
+    dialect: 'postgres'
+});
+
+const SequelizeStore = SequelizeStoreInit(session.Store);
+const sessionStore = new SequelizeStore({
+  db: newSequelize
+});
```

Comment:
Sequelize is an equivalence to prisma. I don't think you are using it for your server though.

Reply:
To do